Explorer/Assets/DCL/Communities/CommunitiesCard/CommunityCardContextMenuConfiguration.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine;
+
+namespace DCL.Communities.CommunitiesCard
+{
+    [CreateAssetMenu(fileName = "CommunityCardContextMenuSettings", menuName = "DCL/Communities/Card/ContextMenuSettings")]
+    [Serializable]
```

Comment:
Serializable is redundant here. All UnityEngine.Object derived types are serializable.